.travis.yml

```diff
@@ -27,3 +27,9 @@ matrix:
       env:
         - MONGOID_VERSION=HEAD
     - rvm: 2.3.0
+
+before_install:
+  - gem update bundler
+
+before_script:
+  - bundle exec danger
```

Comment:
I would move this into one of the matrix items above, for example the last one. There's no need to execute danger NxM times.